### ISSUE TITLE
search: wrong collection triggers file download in browser

### ISSUE THREAD

**Issue by jmartinm** [inveniosoftware/invenio#2940]
How to reproduce
==============

With Invenio demosite and Invenio both on master branch, perform a search like:

```
http://0.0.0.0:5000/search?ln=en&cc=test&p=test&action_search=
```

The 'wrong' ``cc`` parameter makes the browser download a binary file.

BTW, how to create a new collection? on my overlay I created a new one (it is inserted correctly in the ``collection`` table) but I am still seeing this behaviour. That's how I discovered this bug.

**Comment by jirikuncar** [comment]
> Invenio demosite and Invenio both on master branch

I'm changing the milestone to 2.1 then.

**Comment by jirikuncar** [comment]
The problem can be solved by #2958.

**Comment by jirikuncar** [comment]
Closed via #2975.
